app/models/miq_report/generator.rb

```diff
@@ -194,7 +194,9 @@ def _generate_table(options = {})
     if custom_results_method
       if klass.respond_to?(custom_results_method)
         # Use custom method in DB class to get report results if defined
-        results, ext = klass.send(custom_results_method, db_options[:options].merge(:userid => options[:userid], :ext_options => ext_options))
+        results, ext = klass.send(custom_results_method, db_options[:options].merge(:userid      => options[:userid],
+                                                                                    :ext_options => ext_options,
+                                                                                    :report_cols => cols))
```

Comment:
I'm ok with this.

Just a side note / thought:

at one time, `:report_cols` was part of `:ext_options`. Unfortunately, `:ext_options` has been abused for too long and we were making it go away (albeit slowly).

Can we use `:select` or something else that is part of active record's current framework? (Yes, I know this is not an active record call)